modules/ROOT/pages/proxy-deploy-raml-oas-proxy.adoc

```diff
@@ -1,32 +1,45 @@
-= Building a RAML/OAS API Proxy
+= Building a RAML or OAS API Proxy
 
 API Manager 2.x allows you to use a RAML/OAS type of API Proxy, for all your Mule 4 based proxies.
 
-== Configuring your RAML/OAS API Proxy
+== Configuring Your RAML or OAS API Proxy
 
-. In API Manager, navigate to *Manage API* to create your API Instance. +
-You can choose to either xref:manage-exchange-api-task.adoc[Creating an API instance by importing from Exchange] or xref:manage-exchange-api-task.adoc[Creating an API instance by importing on API Manager].
-. In *Asset Type*, select *RAML/OAS*.
-. To configure your API Proxy for this API instance you need to select *Endpoint with Proxy* in the *Managing Type* fields.
-.. Select your *Proxy Deployment Target*  from the available deployment targets displayed.
-.. Select the *Check this box if you are managing this API in Mule 4 or above* check mark.
-.. Type in your *Implementation URI*.
+. In API Manager, navigate to *Manage API* to create your API instance. 
++
+You can create an API instance by importing from Exchange or by importing in API Manager. 
+. From *Asset Type*, select *REST/OAS*.
+. To configure your API proxy for this API instance, select *Endpoint with Proxy* in the *Managing type* field.
+. In the *Proxy Deployment Target* field, select your deployment target for the proxy.
+. Select *Check this box if you are managing this API in Mule 4 or above*.
++
+The strict validation options are automatically displayed under the *Validations* field.
+. In the *Implementation URI* field, type the URI for the backend service.
 +
 image::proxy-deploy-raml-oas-proxy-85634.png[]
 . Click *Advanced Options*.
 . In *Proxy Version*, select *latest*. +
 This value ensures that your API proxy uses the latest released proxy version.
 +
-Versions 2.0.0 and later are the recommended version for OAS/RAML specs, since these versions add native OAS support. +
-If you upload an OAS API specification to an API proxy version 1.0 or earlier, your API specification will be translated to RAML.
-. Select your *Scheme* and *Port* number for your API proxy. +
-Optionally, you can set up an *API Instance label* and a *Response Timeout*.
+Versions 2.0.0 and later are recommended for OAS/RAML specs, since these versions add native OAS support.
```

Comment:
Maybe we can add context to what "versions" word mean by adding "Proxy Version 2.0.0 and later".

Let's check "The strict validation options are automatically displayed" wording since this automatically displays only if the Proxy version has the feature. So if it is not displayed means that the user needs to update the proxy version. (right @FedeAmdan ?)

Reply:
Sergio is right, OAS is supported since 2.0.0, but the the strict validation options are automatically displayed since 2.1.0.. So IMHO I would rephrase it to "From 2.1.0 the regular and strict validation checkboxes are automatically displayed"

Reply:
Done! Thanks!